src/__init__.py

```diff
@@ -332,6 +332,15 @@ def add_filler_items(self, item_pool, traps):
 
         return item_pool
 
+    def get_item_counts(self, player: Optional[int] = None, reset: bool = False) -> dict:
+        """returns the player real item count"""
+        if player == None:
```

Comment:
I left this optionnal player argument for is someone in the futur want to get the counts of another player